extensions/wikia/WikiFactory/Close/maintenance.php

```diff
@@ -90,13 +91,11 @@ public function execute() {
 		);
 
 		$this->info( 'wikis to remove', [
-			'wikis' => $sth->numRows()
+			'wikis' => $sth->numRows(),
+			'query' => $dbr->lastQuery()
 		] );
 
-		$this->log( 'Wikis to remove: ' . $sth->numRows() );
-		$this->log( $dbr->lastQuery() );
-
-		$this->log( 'Will start in 5 seconds...' );
+		$this->info( 'Will start in 5 seconds...' );
```

Comment:
Does it make sense to log this line to Kibana? I'd actually remove this sleep. This script is no longer run manually, so we  do not need to wait for anyone :)